### PR TITLE
add AWS_SES_PORT env

### PR DESCRIPTION
**Pull request by bigpigeon**
ses often connection timeout that used 25 port, therefore I  add a choosesable env

### CODE REVIEW

File: target/start-mailserver.sh [hunk 1]
```diff
@@ -249,8 +249,11 @@ fi
...
 if [ ! -z "$AWS_SES_HOST" -a ! -z "$AWS_SES_USERPASS" ]; then
-  echo "Setting up outgoing email via AWS SES host $AWS_SES_HOST"
-  echo "[$AWS_SES_HOST]:25 $AWS_SES_USERPASS" >>/etc/postfix/sasl_passwd
+  if [ -z "$AWS_SES_PORT" ];then
+    AWS_SES_PORT=25
+  fi
+  echo "Setting up outgoing email via AWS SES host $AWS_SES_HOST:$AWS_SES_PORT"
+  echo "[$AWS_SES_HOST]:$AWS_SES_PORT $AWS_SES_USERPASS" >>/etc/postfix/sasl_passwd
   postconf -e \
     "relayhost = [$AWS_SES_HOST]:25" \
```
Comment (tomav) [inline target/start-mailserver.sh]: We should get AWS_SES_PORT on line 256 too, no?

Comment (bigpigeon) [inline target/start-mailserver.sh]: you also replace 587 with 25th port
use 25th port is bad idea!

Comment (bigpigeon) [inline target/start-mailserver.sh]: http://docs.aws.amazon.com/ses/latest/DeveloperGuide/smtp-connect.html

**Comment by tomav** [comment]
Thanks!

**Pull request merged**